modules/programs/exa.nix

```
@@ -23,10 +11,49 @@ in {
 
     enableAliases = mkEnableOption "recommended exa aliases";
 
+    extraOptions = mkOption {
+      type = types.listOf types.str;
+      default = [ ];
+      example = [ "--group-directories-first" "--header" ];
+      description = ''
+        Extra command line options passed to exa.
+      '';
+    };
+
+    icons = mkOption {
+      type = types.bool;
+      default = false;
+      description = ''
+        Display icons next to file names (`--icons` argument).
+      '';
+    };
+
+    git = mkOption {
+      type = types.bool;
+      default = false;
+      description = ''
+        List each file's Git status, if tracked or ignored (`--git` argument).
+      '';
+    };
+
     package = mkPackageOption pkgs "exa" { };
   };
 
-  config = mkIf cfg.enable {
+  config = let
+    cfg = config.programs.exa;
+
+    baseCmd = concatStringsSep " " ([ "${pkgs.exa}/bin/exa" ]
```

Perhaps this should be
```suggestion
    baseCmd = concatStringsSep " " ([ "${cfg.package}/bin/exa" ]
```
?